test/clt-tests/core/test-comparison-overhead-json-sql.rec

```diff
@@ -0,0 +1,14 @@
+––– block: ../base/start-searchd –––
+––– input –––
+chmod +x ./test/clt-tests/core/script-for-comparison-join-sql.sh
+––– output –––
+––– input –––
+./test/clt-tests/core/script-for-comparison-join-sql.sh 2>/tmp/error.log && grep -oP 'Duration: \K\S+' /tmp/error.log
```

Comment:
Why the commands are not executed directly in the CLT test? Calling a test script is not how CLT was designed.